client/my-sites/sharing/buttons/options.jsx

```diff
@@ -134,13 +134,22 @@ module.exports = React.createClass( {
 			return;
 		}
 
+		const checked = typeof this.props.values.jetpack_comment_likes_enabled === 'undefined'
```

Comment:
I have this habit to avoid using lodash as much as I can, but you're right, it's probably a good habit while writing libraries (and even, we tree shaking we don't have to worry about this so much, so I have to change this :P) but not for applications :)